dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -30,7 +30,7 @@ keystore.desc=A keystore containing key entries for the SSL repertoire. This att
 truststore=Truststore
 truststore.desc=A keystore containing trusted certificate entries used by the SSL repertoire for signing verification. This attribute is optional. If unspecified, the same keystore is used for both key and trusted certificate entries.
 repertoire.sslProtocol=SSL protocol
-repertoire.sslProtocol.desc=The SSL handshake protocol. Protocol values can be found in the documentation for the underlying JRE's Java Secure Socket Extension (JSSE) provider.  When using the IBM JRE the default value is SSL_TLSv2 and when using the Oracle JRE the default value is SSL.
+repertoire.sslProtocol.desc=The SSL handshake protocol. Protocol can be a single value found in the documentation for the underlying JRE's Java Secure Socket Extension (JSSE) provider.  When using the IBM JRE the default value is SSL_TLSv2 and when using the Oracle JRE the default value is SSL.  The protocol can also be a comma separated list of protocol values from TLSv1, TLSv1.1, TLSv1.2, TLSv1.3 values.
```

Comment:
Change:
Protocol can be
to:
The protocol can be set to

Change:
The protocol can also be a comma separated list of protocol values from TLSv1, TLSv1.1, TLSv1.2, TLSv1.3 values.
to
The protocol can also be a comma-separated list of any of the following values: TLSv1, TLSv1.1, TLSv1.2, or TLSv1.3.

Reply:
@jantley-ibm  The message has been updated please review.

Reply:
@acdemyers I see the second change but not the first, was that intentional?

Reply:
@jantley-ibm Sorry my error.  I missed putting that.  It's been updated please review again.

Reply:
@acdemyers looks good, thanks! I added the ID reviewed tag.